smart-contracts/contracts/cl-vault/src/vault/any_deposit.rs

```diff
@@ -308,28 +322,35 @@ fn swap_msg_token_in_out_amounts(
     let token_out_min_amount = token_out_ideal_amount?
         .checked_multiply_ratio(max_slippage.numerator(), max_slippage.denominator())?;
 
+    if !pool_config.pool_contains_token(token_in_denom) {
+        return Err(ContractError::InvalidSwapAssets {});
+        // TODO: Restore this error
```

Comment:
What's the issue that we don't have this anymore, but need it back?

Reply:
put back. also on range.rs